app/order-history/page.tsx

```diff
@@ -0,0 +1,381 @@
+            <optgroup label="Order Statuses">
+              {statuses.map((status) => (
+                <option key={status} value={status} className="text-black">{status}</option>
+              ))}
+            </optgroup>
+            <optgroup label="Payment Statuses">
+              {paymentStatuses.map((paymentStatus) => (
+                <option key={paymentStatus} value={paymentStatus} className="text-black">{paymentStatus}</option>
+              ))}
+            </optgroup>
+          </select>
+        </div>
+      </div>
+
+      <div className="flex flex-wrap gap-2 mt-4">
+        {/* Filter badges */}
+        {serviceFilter.map((filterType, index) => (
+          <div
+            key={index}
+            className="flex items-center bg-gray-200 px-3 py-1 rounded-full text-gray-700"
+          >
+            {filterType}
+            <button
+              onClick={() => setServiceFilter((prev) => prev.filter((type) => type !== filterType))}
+              className="ml-2 text-gray-500 hover:text-gray-700"
+            >
+              <X className="h-4 w-4" />
+            </button>
+          </div>
+        ))}
+
+        {logisticsFilter && (
+          <div
+            className="flex items-center bg-gray-200 px-3 py-1 rounded-full text-gray-700"
+          >
+            {logisticsFilter}
+            <button
+              onClick={() => setLogisticsFilter(null)}
+              className="ml-2 text-gray-500 hover:text-gray-700"
+            >
+              <X className="h-4 w-4" />
+            </button>
+          </div>
+        )}
+
+        {statusFilter && (
+          <div
+            className="flex items-center bg-gray-200 px-3 py-1 rounded-full text-gray-700"
+          >
+            {statusFilter}
+            <button
+              onClick={() => setStatusFilter(null)}
+              className="ml-2 text-gray-500 hover:text-gray-700"
+            >
+              <X className="h-4 w-4" />
+            </button>
+          </div>
+        )}
+
+        {paymentStatusFilter && (
+          <div
+            className="flex items-center bg-gray-200 px-3 py-1 rounded-full text-gray-700"
+          >
+            {paymentStatusFilter}
+            <button
+              onClick={() => setPaymentStatusFilter(null)}
+              className="ml-2 text-gray-500 hover:text-gray-700"
+            >
+              <X className="h-4 w-4" />
+            </button>
+          </div>
+        )}
+
+        {/* Clear all filters Button */}
+        <button
+          onClick={clearFilters}
+          disabled={!filter && serviceFilter.length === 0 && !logisticsFilter && !statusFilter && !paymentStatusFilter}
+          className={`ml-2 px-4 py-2 rounded-lg transition-all ${filter || serviceFilter.length > 0 || logisticsFilter || statusFilter || paymentStatusFilter ? "bg-[#9CC0D9] text-white hover:bg-blue-700" : "bg-gray-300 text-gray-500 cursor-not-allowed"}`}
+        >
+          Clear All
+        </button>
```

Comment:
you could have used shadcns button. as much as possible use shadcns component so that uniform and standardized lahat ng pages.